kolibri/plugins/device_management/assets/src/userSearchUtils.js

```diff
@@ -8,5 +6,8 @@ export function userMatchesFilter(user, searchFilter) {
 }
 
 export function filterAndSortUsers(users, pred) {
-  return orderBy(users.filter(pred), [user => user.username.toUpperCase()], ['asc']);
+  return users.filter(pred).sort(
```

Comment:
could you integrate this into the 'learner-list-page' (list is not alphabetized)? the `pred` would just be identity function since there's no filter.